lhotse/kaldi.py

```diff
@@ -42,7 +44,7 @@ def get_duration(
         import soundfile
 
         info = soundfile.info(path)
-    except:
+    except ImportError:
```

Comment:
I don't think this is right -- the idea of this logic is to go from most-adequate/fastest backend -> most-flexible/slowest backend. With audio it seems you can never predict whether a given backend will be able to decode a given file type (missing/non-typical headers, format options in WAV/FLAC/SPHERE/OPUS/etc.), so if they fail on anything, we just try to go to the next one. 

Can you revert this change?

Reply:
Reverted.